.gitignore

```diff
@@ -18,3 +18,4 @@
 
 .DS_Store
 .env
+config/database.yml
```

Comment:
This is b/c my postgres on linux defaults to expect a username for some reason. Even after setting the trust authentications.